.github/workflows/npm-publish-agent-bus.yml

```
@@ -0,0 +1,62 @@
+name: publish-agent-bus
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)